File: apps/android/app/src/main/java/com/psp2/plugin/my-second/MySecondPlugin.kt
package com.psp2.mysecondplugin

import com.getcapacitor.JSObject
import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "MySecond")
class MySecondPlugin : Plugin() {
    @PluginMethod
    fun logMessageee(call: PluginCall) {
        // 從前端獲取訊息
        val message = call.getString("message") ?: "No message provided"

        val ret = JSObject()
        ret.put("message", message)

        // 回應前端呼叫
        call.resolve(ret)
    }
}

File: apps/ios/App/App/Plugin/my-second/MySecondPlugin.swift
import Capacitor

@objc(MySecondPlugin)
public class MySecondPlugin: CAPPlugin, CAPBridgedPlugin {
    public let identifier = "MySecondPlugin"
    public let jsName = "MySecond"
    public let pluginMethods: [CAPPluginMethod] = [
        CAPPluginMethod(name: "logMessageee", returnType: CAPPluginReturnPromise)
    ]

    @objc func logMessageee(_ call: CAPPluginCall) {
        // 從前端獲取訊息
        let message = call.getString("message") ?? "No message provided"

        // 回應前端呼叫
        call.resolve(["message": message])
    }
}
